### PR TITLE
[51] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by EseAlli**
Develop a Contact Form Page using HTML
Created a contact form page

### CODE REVIEW

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]
Hello @Ese,

Good job on your PR!
Please attach a screenshot of the implementation.

File: submissions/alliEse/html-forms/contact.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	<meta charset="utf-8">
+	<title>Contact Us - Brogues by Wingtip Brogues</title>
+</head>
+<body>
+<header>
+        <h1>Brogues by Wingtip Brogues</h1>
+</header>
+    <nav>
+        <ul>
+                <li><a href="#"> Home</a></li>
```
Comment (emmanuelidiaghe) [inline submissions/alliEse/html-forms/contact.html:13]: Please remove the href attribute and its value from all a-tags in the nav. The linking will be done in subsequent tasks.

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]

File: submissions/alliEse/HTML/index.html [hunk 2]
```diff
@@ -6,7 +6,14 @@
     <title>Essential Fantasies - Wingtip Brogues</title>
   </head>
   <body>
-    <nav><h1>Essential Fantasies</h1></nav>
+    <nav><h1>Essential Fantasies</h1>
+    <ul>
+				<li><a href="#"> Home</a></li>
+				<li><a href="#">Blog</a> </li>
+				<li><a href="index.html">Productpage</a> </li>
+				<li><a href="../html-forms/contact.html">Contact Us</a> </li>
```
Comment (adaugochi) [inline submissions/alliEse/HTML/index.html:14]: Well done @EseAlli 

Linking of pages will be done in your next task. You can remove them for now

File: submissions/alliEse/html-forms/contact.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                <li><a href="../HTML/index.html">Product</a> </li>
```
Comment (adaugochi) [inline submissions/alliEse/html-forms/contact.html:14]: Same here too

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/alliEse/html-forms/contact.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <label for="subject">Subject:</label>
+            <input type="text" name="subject" id="subject" placeholder="Enter A Subject" required/> <br>
+            <br>
+            <label for="message">Message:</label>
+            <textarea name="message" id="message" rows="10" cols="100" placeholder="Please Drop Your Message"></textarea> <br>
+            <br>
+			<button type="submit">Submit</button>
+	</fieldset>
+<footer>
+	<p> Find us here:
```
Comment (ToluAkin) [inline submissions/alliEse/html-forms/contact.html:37]: The p tag should close on this line as well @EseAlli

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by EseAlli** [comment]
image

**Pull request merged**